src/databricks/labs/mcp/servers/unity_catalog/server.py

```diff
@@ -44,23 +52,105 @@ def get_tools_dict(settings) -> dict[str, BaseTool]:
     }
 
 
-async def start() -> None:
-    server = Server(name="mcp-unitycatalog", version=VERSION)
+def start() -> None:
+    # server = Server(name="mcp-unitycatalog", version=VERSION)
+    server = FastMCP(name="mcp-unitycatalog", version=VERSION)
     tools_dict = get_tools_dict(settings=get_settings())
 
-    @server.list_tools()
-    async def list_tools() -> list[ToolSpec]:
-        return [tool.tool_spec for tool in tools_dict.values()]
 
-    @server.call_tool()
-    async def call_tool(name: str, arguments: dict) -> list[Content]:
-        tool = tools_dict[name]
-        return tool.execute(**arguments)
+    
+    @server.tool()
+    def run_sql(sql: str) -> str:
+        """
+        Run a stateless, read-only SQL query in Databricks.
+        
+        Can be used to also query metadata, e.g. 
```

Comment:
Putting this stuff in the docstring (which is automatically detected by `@mcp.tool()` worked pretty well